src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -1328,11 +1365,27 @@ private static object SearchIntrinsicTable(Hashtable table, Type callingType)
 
                 if (type != null)
                 {
-                    hashEntry = CreateInstance(type, callingType);
                     if (type.GetConstructor(s_typeConstructor) == null)
```

Comment:
Can this be changed to check if the entry exists in the dictionary before performing reflection:

```C#
if (ConverterConstructorFuncs.TryGetvalue(type, out ...)
{
   
}
```